Extensions/Firebase/B_firebasetools/D_authtools.ts

```diff
@@ -41,7 +41,7 @@ namespace gdjs {
         /**
          * The current authentication status.
          */
-        export let authentified = false;
+        export let authenticated = false;
```

Comment:
This one, you shouldn't have to keep the previous ill-named property since there's the getter below. Even if the property is exported.